source/_integrations/unifiprotect.markdown

```diff
@@ -182,3 +194,11 @@ Unlike with many other things, playing audio to your speakers requires your Home
 ### Liveview Options for Viewport Missing Options or Out of Date
 
 Main control selects currently cannot have dynamic options since the options are exported out to voice assistants. After you add/remove/change a Liveview in UniFi Protect, you must restart Home Assistant to get the new options for your Viewport.
+
+### NvrErrors with "404 - Reason: Not Found"
+
+If you get errors while authenticating or fetching data for `NvrError... 404 - Reason: Not Found`, there is a good chance that your UniFi Protect application has crashed. UniFi Protect runs in a supervised way on UniFi OS (similar to Home Assistant OS + Home Assistant Core). Getting a 404 for a URL that should not 404 means UniFi Protect is probably not running. You may want to check the health of your disks or look into debugging UniFi Protect to see why it is crashing.
```

Comment:
```suggestion
If you get errors while authenticating or fetching data for `NvrError... 404 - Reason: Not Found`, there is a good chance that your UniFi Protect application has crashed. UniFi Protect runs in a supervised way on UniFi OS (similar to Home Assistant OS + Home Assistant Core). Getting a 404 for a URL that should that should not produce a 404 means UniFi Protect is probably not running. You may want to check the health of your disks or look into debugging UniFi Protect to see why it is crashing.
```

Reply:
I think you have two `that should that should` in there.

Reply:
Should we wrap the 404 message and re-throw the error with this information in the lib?

Reply:
I mean we could, but then how do you know if it is a real 404 or just because protect is not running? There would need to be a good way to tell them apart (future problem basically).

Reply:
You could also get a 404 on doing a GET to a camera you do not have permission to see, for example.

Reply:
Fair enough.  We don't know enough to give them a good error in this case.